managed/services/management/backup/backups_service.go

```diff
@@ -114,7 +114,14 @@ func (s *BackupsService) StartBackup(ctx context.Context, req *backupv1beta1.Sta
 	case models.MySQLServiceType:
 		dataModel = models.PhysicalDataModel
 	case models.MongoDBServiceType:
-		dataModel = models.LogicalDataModel
+		switch req.DataModel {
+		case backupv1beta1.DataModel_LOGICAL:
+			dataModel = models.LogicalDataModel
+		case backupv1beta1.DataModel_PHYSICAL:
+			dataModel = models.PhysicalDataModel
+		default:
+			return nil, status.Errorf(codes.InvalidArgument, "unknown data model: %s", req.DataModel.String())
```

Comment:
Not sure what a good name for a `convert*` function here will look like (`convertModelToBackupModel` is confusing), so I'm opting to duplicate the code for now.

Reply:
Better to create function with not so good name, but not duplicate code. Let's use `convertModelToBackupModel` for now, and refactor it later.